### PR TITLE
[WebApp] Edit passkey name 

### PR DESCRIPTION
**Pull request by Maks19**
**Description:** Edit passkey name 

**Issue:** https://www.notion.so/saladtech/FE-Edit-passkey-name-bf4f9a16515a450f9752e0cb350f084a

image

### CODE REVIEW

File: packages/web-app/src/modules/passkey-setup/PasskeyStore.tsx [hunk 1]
```diff
@@ -175,8 +181,22 @@ export class PasskeyStore {
     this.hasVerifyWithPasskeyFailed = updatedHasVerifyWithPasskeyFailedFailed
   }
 
-  @action.bound
```
Comment (vitto-moz) [inline packages/web-app/src/modules/passkey-setup/PasskeyStore.tsx:178]: don't remove this line it leads to the lost context of mobx store in `setRegisterPasskeyStatus`

**Review by vitto-moz — CHANGES_REQUESTED** [review]

**Comment by vitto-moz** [comment]
long passkey name intersects with a date
image

**Comment by vitto-moz** [comment]
the submit button looks not to be aligned with a rest of UI
image

**Comment by vitto-moz** [comment]
After the first submit there are no input controls on focus and editing (there are no submit and undo icons)
image

File: packages/web-app/src/modules/passkey-setup/PasskeyStore.tsx [hunk 2]
```diff
@@ -32,6 +32,12 @@ export class PasskeyStore {
   @observable
   public hasVerifyWithPasskeyFailed: boolean = false
 
+  @observable
+  public isEditPasskeySubmitting: boolean = false
+
+  @observable
+  public isEditPasskeyNameSubmitSuccess: boolean = false
+
```
[the remark this answers is not in the document]
Comment (Maks19) [reply packages/web-app/src/modules/passkey-setup/PasskeyStore.tsx:40]: When do we need `unknown` status?

**Comment by Maks19** [comment]
> there is a unnecessary space on the input focus image

It's necessary because there are two icons
I used the same input as we have for other usernames

File: packages/web-app/src/modules/passkey-setup/PasskeyStore.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (vitto-moz) [reply packages/web-app/src/modules/passkey-setup/PasskeyStore.tsx:40]: `unknown` is the initial status - before the submitting process started.

File: packages/web-app/src/modules/passkey-setup/PasskeyStore.tsx (hunk 1 above)
Comment (vitto-moz) [reply packages/web-app/src/modules/passkey-setup/PasskeyStore.tsx:178]: fixed

**Comment by vitto-moz** [comment]
@Maks19 I've added commits to cover all the mentioned issues - please, check and we can discuss if needed

**Review by vitto-moz — DISMISSED** [review]

**Pull request merged**